### ISSUE TITLE
[bug] GLIBC not found with version 4.1.3 on ubuntu 20.04 focal

### ISSUE THREAD

**Issue by GregOriol** [centrifugal/centrifugo#644]
Hi,

On ubuntu 20.04, centrifugo version 4.1.3 installed via apt with the debs provided at `https://packagecloud.io/FZambia/centrifugo` (following the installation instructions here: `https://centrifugal.dev/docs/getting-started/installation#rpm-and-deb-packages-for-linux`) seem to have an issue when trying to run it:
```
$ centrifugo -c /etc/centrifugo/config.toml
centrifugo: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by centrifugo)
centrifugo: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by centrifugo)
```

Centrifugo version 4.1.2 installed the same way works well on the same system.

System is:
```
$ uname -a
Linux [...] 5.4.0-139-generic #156-Ubuntu SMP Fri Jan 20 17:27:18 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux
$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 20.04.6 LTS
Release:	20.04
Codename:	focal
$ ldd --version
ldd (Ubuntu GLIBC 2.31-0ubuntu9.9) 2.31
```

Is is possible that the build process has changed and uses a more recent version of ubuntu which tries to link glibc with a newer version?

**Comment by FZambia** [comment]
Hello @GregOriol 

Unfortunate :( Seems like it's common trouble within the ecosystem:

* https://github.com/golang/go/issues/57328
* https://github.com/golang/go/issues/58550

The solution seems to be using `CGO_ENABLED=0` in our deb/rpm build process. Could you please verify that the binary release  `centrifugo_4.1.3_linux_amd64.tar.gz`:

```
https://github.com/centrifugal/centrifugo/releases/download/v4.1.3/centrifugo_4.1.3_linux_amd64.tar.gz
```

– loaded from https://github.com/centrifugal/centrifugo/releases/tag/v4.1.3 page works fine on your Ubuntu server (for those binaries CGO_ENABLED=0 is [already used](https://github.com/centrifugal/centrifugo/blob/master/.goreleaser.yml#L58))

**Comment by GregOriol** [comment]
@FZambia Yeah I don't know much about Go, but I've found the same links before opening the issue.

The file you link to `https://github.com/centrifugal/centrifugo/releases/download/v4.1.3/centrifugo_4.1.3_linux_amd64.tar.gz` works fine! Seems that `CGO_ENABLED=0` is indeed the solution to this problem.

**Comment by FZambia** [comment]
Fixed the build script, also manually triggered re-build and upload of DEB/RPM packages with the next iteration number - see [on top of the list](https://packagecloud.io/FZambia/centrifugo) - `4.1.3-1`. @GregOriol – could you please try it ?

**Comment by GregOriol** [comment]
@FZambia It's good with version `4.1.3-1` :-)

**Comment by FZambia** [comment]
Thanks for the report!
